packages/plugin-lighthouse/src/lib/utils.unit.test.ts

```diff
@@ -121,684 +112,250 @@ describe('validateOnlyCategories', () => {
+  });
+
+  it('should return given audits and groups if no filter is set', () => {
+    const audits = [{ slug: 'speed-index' }] as Audit[];
+    const groups = [
       {
         slug: 'performance',
-        title: 'Performance',
-        refs: [{ slug: 'speed-index', weight: 1 }],
+        refs: [{ slug: 'speed-index' }],
       },
-    ];
+    ] as Group[];
     const { audits: filteredAudits, groups: filteredGroups } =
       filterAuditsAndGroupsByOnlyOptions(audits, groups, {});
 
     expect(filteredAudits).toStrictEqual(audits);
     expect(filteredGroups).toStrictEqual(groups);
 
-    const pluginConfig: PluginConfig = {
-      slug: 'coverage',
-      title: 'Code Coverage',
-      icon: 'file',
-      description: 'Runs test coverage and created audits',
-      audits: filteredAudits,
-      groups: filteredGroups,
-      runner: {
-        command: 'node',
-        outputFile: 'out.json',
-      },
-    };
-
+    const pluginConfig = basePluginConfig(filteredAudits, filteredGroups);
     expect(() => pluginConfigSchema.parse(pluginConfig)).not.toThrow();
   });
 
-  it('should filter audits if onlyAudits is set', () => {
+  it('should filter audits if skipAudits is set', () => {
     const { audits: filteredAudits, groups: filteredGroups } =
       filterAuditsAndGroupsByOnlyOptions(
         [
-          { slug: 'speed-index', title: 'Speed Index' },
-          { slug: 'first-contentful-paint', title: 'First Contentful Paint' },
-        ],
+          { slug: 'speed-index' },
+          { slug: 'first-contentful-paint' },
+        ] as Audit[],
         [
           {
             slug: 'performance',
-            title: 'Performance',
-            refs: [{ slug: 'speed-index', weight: 1 }],
+            refs: [{ slug: 'speed-index' }, { slug: 'first-contentful-paint' }],
           },
-        ],
-        { onlyAudits: ['speed-index'] },
+        ] as Group[],
+        { skipAudits: ['speed-index'] },
       );
 
-    expect(filteredAudits).toStrictEqual<Audit[]>([
-      { slug: 'speed-index', title: 'Speed Index' },
-    ]);
-    expect(filteredGroups).toStrictEqual<Group[]>([
-      {
-        slug: 'performance',
-        title: 'Performance',
-        refs: [{ slug: 'speed-index', weight: 1 }],
-      },
-    ]);
-
-    const pluginConfig: PluginConfig = {
-      slug: 'coverage',
-      title: 'Code Coverage',
-      icon: 'file',
-      description: 'Runs test coverage and created audits',
-      audits: filteredAudits,
-      groups: filteredGroups,
-      runner: {
-        command: 'node',
-        outputFile: 'out.json',
-      },
-    };
+    expect(filteredAudits).toHaveLength(1);
+    expect(filteredAudits.at(0)?.slug).toBe('first-contentful-paint');
+
+    expect(filteredGroups).toHaveLength(1);
+    expect(filteredGroups.at(0)?.slug).toBe('performance');
+    expect(filteredGroups.at(0)?.refs).toHaveLength(1);
+    expect(filteredGroups.at(0)?.refs.at(0)?.slug).toBe(
+      'first-contentful-paint',
+    );
```

Comment:
I do not recall advocating for nested conditional chaining in tests where the object structure should only have one expected value. If you can find the comment, please send it.

The approach below cleanly avoids the TypeScript complaints. Same approach to be used in other applicable cases.

```suggestion
    expect(filteredAudits).toStrictEqual([{ slug: 'first-contentful-paint' }]);
    expect(filteredGroups).toStrictEqual([
      {
        slug: 'performance',
        refs: [{ slug: 'first-contentful-paint' }],
      },
    ]);
```